internal/component/configmap/configmap_test.go

```diff
@@ -377,8 +370,20 @@ func matchClientTLSRelatedConfiguration(g *WithT, etcd *druidv1alpha1.Etcd, actu
 	}
 }
 
+func getAdvertiseURLsInterface(etcd *druidv1alpha1.Etcd, commType, scheme string) map[string]interface{} {
```

Comment:
A few comments:

1. What is the need to create a type alias `type advertiseURLs map[string][]string`? There is no method defined on this type. If its only for readability then that is a not a strong argument to have it.
2. Can you please rename this function to `convertAdvertiseURLsValuesToInterface`

Reply:
1. Yes it was only for readability. Will remove it.
2. Sure, this is more readable.
Thanks

Reply:
Done